doc/examples/nonstationary_inference.rst

```diff
@@ -0,0 +1,87 @@
+.. jupyter-execute::
+    :hide-code:
+
+    import set_working_directory
+
+
+.. _nonstationary-model-aa-inference:
+
+
+Infering Amino Acid frequencies with a nonstationary model
+==========================================================
+
+.. sectionauthor:: Peter Goodman, Andrew Wheeler
+
+This example demonstrates how to apply a non-stationary, branch-heterogeneous maximum likelihood model to infer site frequency vectors across different branches or branch groups on a phylogenetic tree, highlighting a practical use case. The model used is a maximum likelihood, time-heterogeneous, and time-reversible model.
+
+Non-stationarity means that amino acid frequencies are not fixed over genetic time. This characteristic allows a non-stationary model to infer new frequencies along different branches or clades of a tree.
+
+In this example, we infer an amino acid frequency vector for the primate and rodent clades in a mammal phylogeny, as well as a frequency vector for the root of these two clades. The purpose of this inference was to produce three amino acid frequency vectors for comparison. The rodent vector represented the amino acid frequencies associated with a higher effective population size (:math:`N_{e}`), while the primate vector showed frequencies associated with a lower :math:`N_{e}`.  In a related analysis, we examined which amino acids were preferred in species with higher :math:`N_{e}` values and correlated those preferences with other amino acid fitness metrics.
```

Comment:
"associated with a higher effective population size" change to "associated with a clade reported to exhibit higher effective population sizes", and make the corresponding change "reported to exhibit lower ..." for the primate statement.
As you're not including the other analyses implied by "In a related analysis, we examined which amino acids were preferred", I suggest editing to being something along the lines of "These clade specific measures allow measuring associations with other known properties of the clades", or something like that.